packages/augur-sdk/src/state/db/BlockAndLogStreamerListener.ts

```diff
@@ -1,77 +1,122 @@
-import { EthersProviderBlockStreamAdapter, ExtendedLog } from "blockstream-adapters";
-import { Block, BlockAndLogStreamer, Log as BlockStreamLog } from "ethereumjs-blockstream";
-import { Filter } from "ethereumjs-blockstream/output/source/models/filters";
-import { EthersProvider } from "@augurproject/ethersjs-provider";
-import { Log, ParsedLog } from "@augurproject/types";
-import { EventLogDBRouter } from "./EventLogDBRouter";
-
-export interface BlockAndLogStreamerInterface<TBlock extends Block, TLog extends BlockStreamLog> {
+import {
+  EthersProviderBlockStreamAdapter,
+  ExtendedLog,
+} from 'blockstream-adapters';
+import {
+  Block,
+  BlockAndLogStreamer,
+  Log as BlockStreamLog,
+} from 'ethereumjs-blockstream';
+import { Filter } from 'ethereumjs-blockstream/output/source/models/filters';
+import { EthersProvider } from '@augurproject/ethersjs-provider';
+import { Log, ParsedLog } from '@augurproject/types';
+import { EventLogDBRouter } from './EventLogDBRouter';
+
+export interface BlockAndLogStreamerInterface<
+  TBlock extends Block,
+  TLog extends BlockStreamLog
+> {
   reconcileNewBlock: (block: TBlock) => Promise<void>;
   addLogFilter: (filter: Filter) => string;
   subscribeToOnBlockAdded: (onBlockAdded: (block: TBlock) => void) => string;
-  subscribeToOnBlockRemoved: (onBlockRemoved: (block: TBlock) => void) => string;
-  subscribeToOnLogsAdded: (onLogsAdded: (blockHash: string, logs: TLog[]) => void) => string;
-  subscribeToOnLogsRemoved: (onLogsRemoved: (blockHash: string, logs: TLog[]) => void) => string;
+  subscribeToOnBlockRemoved: (
+    onBlockRemoved: (block: TBlock) => void
+  ) => string;
+  subscribeToOnLogsAdded: (
+    onLogsAdded: (blockHash: string, logs: TLog[]) => void
+  ) => string;
+  subscribeToOnLogsRemoved: (
+    onLogsRemoved: (blockHash: string, logs: TLog[]) => void
+  ) => string;
 }
 
 export interface BlockAndLogStreamerListenerDependencies {
   address: string;
   blockAndLogStreamer: BlockAndLogStreamerInterface<Block, ExtendedLog>;
   getBlockByHash: (hashOrTag: string) => Promise<Block>;
   eventLogDBRouter: EventLogDBRouter;
-  getEventTopics: (eventName: string) => Array<string>;
+  getEventTopics: (eventName: string) => string[];
```

Comment:
I read at some point somewhere that `Array<string>` is the preferred/recommended syntax here, `string[]` is supported for legacy/backward compatibility reasons or something.